package/aistore/client/api.py

```diff
@@ -27,6 +28,27 @@ def __init__(self, url):
     def base_url(self):
         return self._base_url
 
+    def list_buckets(self, provider):
+        url = "{}/buckets".format(self.base_url)
+        params = {}
+        if provider != "":
+            params[URL_PARAM_PROVIDER] = provider
+
+        action = {"action": "list", "value": {}}
+        return requests.get(url=url, headers={"Accept": "application/json"}, json=action, params=params)
+
+    def create_bucket(self, bck_name):
+        url = "{}/buckets/{}".format(self.base_url, bck_name)
+        params = {URL_PARAM_PROVIDER: PROVIDER_AIS}
+        action = {"action": "create-bck", "value": {}}
```

Comment:
Could you change this to use [ActionMsg](https://github.com/NVIDIA/ais-etl/blob/master/package/aistore/client/msg.py#L33) instead of creating `dict` directly?

Reply:
Agreed. It makes sense. I just copied from existing calls. So, I have to rewrite old calls as well.